hack/test-extended/ha/run.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -o nounset
+set -o pipefail
+
+CONTROLLER_LEASE_TTL=10
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/../../..
+cd "${OS_ROOT}"
+
+source ${OS_ROOT}/hack/util.sh
+source ${OS_ROOT}/hack/common.sh
+
+set -e
+ensure_ginkgo_or_die
+set +e
+
+os::build::extended
+
+ensure_iptables_or_die
+
+function cleanup()
+{
+	out=$?
+	cleanup_openshift
+	echo "[INFO] Exiting"
+	exit $out
+}
+
+echo "[INFO] Starting 'ha' extended tests"
+
+trap "exit" INT TERM
+trap "cleanup" EXIT
+
+export TMPDIR="${TMPDIR:-"/tmp"}"
+export BASETMPDIR="${TMPDIR}/openshift-extended-tests/images"
+setup_env_vars
+export MASTER_CONFIG_PATH="${MASTER_CONFIG_DIR}/master-config.yaml"
+reset_tmp_dir 
+
+configure_os_server
+sed -i "s/\(\<controllerLeaseTTL\s*:\s*\)[0-9]\+/\1$CONTROLLER_LEASE_TTL/" \
```

Comment:
Would be nice. In this case `yq` would be more appropriate tough :-). It made me realize that I make an assumption about master config being yaml. I guess the override could be passed to `configure_os_server` like this:

    configure_os_server controllerLeaseTTL=XY other.nested.var=YZ 'some.string.var="foo"'

which would handle dirty work. In that case `jq` would really come handy.